dist/Time-HiRes/t/Watchdog.pm

```diff
@@ -10,30 +10,30 @@ my $watchdog_pid;
 my $TheEnd;
 
 if ($Config{d_fork}) {
-    print("# I am the main process $$, starting the watchdog process...\n");
```

Comment:
Time::HiRes depends on Test::More without a version number, so note() might not be available (unless you update the dependency in Makefile.PL to say "0.84")

Reply:
That seems reasonable, as that's been available since Oct 2008.  p.r. updated.